syntax = "proto3";

package app.api_validation_error_model;

import "google/protobuf/any.proto";

message ValidationError {
    string field = 1;
    string type = 2;
    string message = 3;
    string error_code = 4;
    google.protobuf.Any data = 5;
}

message ValidationErrorResponse {
    repeated ValidationError validation_errors = 1;
}